.github/workflows/centos7-ci.yml

```diff
@@ -0,0 +1,96 @@
+name: CI Centos7
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test_apisix:
+    name: run ci on centos7
+    runs-on: ubuntu-latest
+
+    services:
+      etcd:
+        image: bitnami/etcd:3.4.0
+        ports:
+          - 2379:2379
+          - 2380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
+
+      old_etcd:
+        image: bitnami/etcd:3.3.8
+        ports:
+          - 3379:2379
+          - 3380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
```

Comment:
I still haven't found a place to use these two etcd clusters

Reply:
for ci `centos7`, it should be useless.

it only was used in https://github.com/apache/apisix/blob/924a30db36268ee1d7fab255ddebfc23a7110314/.travis/apisix_cli_test/test_ci_only.sh#L31

Reply:
I am not sure if there are other similar codes

Reply:
ok, I have removed the `old_etcd ` part.

Reply:
ping @moonming @membphis

Reply:
LGTM